README.md

```diff
@@ -18,10 +18,21 @@ A traditional infrastructure-heavy staking integration can take months. Coinbase
 
 Prerequisite: [Node 20+](https://www.npmjs.com/package/node/v/20.11.1)
 
-1. Install this package: `npm install @coinbase/staking-client-library-ts`
+1. Navigate to your project directory where you would like to set up this client, and install it via:
```

Comment:
```suggestion
1. In a fresh directory, run:
```

Reply:
ya makes sense - don't mind pulling this change in, but my idea was developers could be integrating this into existing projects. But assuming they are going through a quick start, they just want to try something quick - so a fresh dir is a good idea.